app/src/main/java/com/framgia/englishconversation/data/source/remote/timeline/TimelineRemoteDataSource.java

```diff
@@ -219,10 +253,15 @@ public void removeListener() {
     }
 
     public List<TimelineModel> getTimelineData(DataSnapshot dataSnapshot,
-            TimelineModel lastTimeline, UserModel userModel) {
+                                               TimelineModel lastTimeline, UserModel userModel) {
         List<TimelineModel> timelineModels = new ArrayList<>();
         for (DataSnapshot snapshot : dataSnapshot.getChildren()) {
             TimelineModel timelineModel = snapshot.getValue(TimelineModel.class);
+            if (timelineModel.getStatusModel() != null) {
+                if (timelineModel.getStatusModel().getStatus() == Status.DELETE) {
```

Comment:
WARNING: [CollapsibleIfStatements] These nested if statements could be combined
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/basic.html#CollapsibleIfStatements

Reply:
pls use &&